contrib/pod-master/podmaster.go

```diff
@@ -35,14 +35,15 @@ import (
 )
```

Comment:
We aren't replacing podmaster are we?  Should these changes be in a separate file?